### ISSUE TITLE
Yet another small uBlock filter list breaks https://edition.cnn.com/

### ISSUE THREAD

**Issue by WoodpeckerBaby** [mtxadmin/ublock#5]
Yet another small uBlock filter list breaks https://edition.cnn.com/

**Comment by mtxadmin** [comment]
What is broken there? I did not see any issues.

(About a month ago there was one mistake, but now all displays well)

**Comment by mtxadmin** [comment]
I noticed one strange thing: in most of my test browsers https://edition.cnn.com/ is ok, but in one it is definitely broken. Looking at Logger, they fired off completely different set of counters and banners (by content supplied from the server, uBlock filters are equal there). Maybe it is the case?

Needs investigating more, looks very unusual.

**Comment by mtxadmin** [comment]
Are cookies are disabled in your browser?


It appears, when cookies are disabled, one of cnn.com scripts breaks the page. 
If one of scripts from verticals-static.cnn.com is disabled, the page is ok even with disabled cookies, but photos are blurry. If all scripts enabled and cookies disabled, the page brakes itself after loading.

**Comment by aravindvnair99** [comment]
@mtxadmin Works fine here with your hosts file.

**Comment by mtxadmin** [comment]
Under some circumstances page can really break itself (see details above).

@WoodpeckerBaby Could you please check again.

**Comment by aravindvnair99** [comment]
> Under some circumstances page can really break itself (see details above).
> 
> @WoodpeckerBaby Could you please check again.

@mtxadmin Yes, for me I tested with a normal fresh install browser with your latest hosts file and it works. So I meant to say the hosts file isn't the issue. Everything else is beyond the scope of your hosts file.

**Comment by mtxadmin** [comment]
By the way, topic starter was used list and not hosts file.

**Comment by aravindvnair99** [comment]
> By the way, topic starter was used list and not hosts file.

Ahh right @mtxadmin, I misread. 👍
